client/src/hooks/datafiles/mutations/useExtract.ts

```diff
@@ -42,14 +42,14 @@ function useExtract() {
   const { mutateAsync } = useMutation({ mutationFn: submitJobUtil });
 
   const extract = ({ file }: { file: TTapisFile }) => {
-    dispatch({
-      type: 'DATA_FILES_SET_OPERATION_STATUS',
-      payload: { status: 'RUNNING', operation: 'extract' },
-    });
-    if (!allocationForExtract) {
-      throw new Error('You need an allocation to extract.', {
-        cause: 'extractError',
+    setStatus({ type: 'RUNNING' });
+
+    if (!false) {
```

Comment:
This seems like it was intended to have a variable, maybe `allocationForExtract`?